packages/node/src/lib/subscribers/subscribers.ts

```diff
@@ -110,27 +129,53 @@ export class Subscribers extends WithHttp implements ISubscribers {
     );
   }
 
+  /**
+   * deprecated use markInAppMessageAs instead
+   */
   async markMessageSeen(subscriberId: string, messageId: string) {
-    return await this.http.post(
+    return await this.novu.post(
       `/subscribers/${subscriberId}/messages/markAs`,
       { messageId, mark: { seen: true } }
     );
   }
 
+  /**
+   * deprecated use markInAppMessageAs instead
+   */
   async markMessageRead(subscriberId: string, messageId: string) {
-    return await this.http.post(
+    return await this.novu.post(
       `/subscribers/${subscriberId}/messages/markAs`,
       { messageId, mark: { read: true } }
     );
   }
 
+  async markInAppMessageAs(
+    subscriberId: string,
+    messageId: string,
+    mark: IMarkFields
+  ) {
+    if (!mark.read && !mark.seen) {
+      throw new Error('Either seen or read must be set');
+    }
```

Comment:
If we are going to add the mark as unread / unseen soon, this will not be a good enough check. Maybe you could check if mark is empty for now. Until we have logic for what makes sense for them to put in the object (like cases where they set both read and seen that conflicts with each other)

Reply:
@ainouzgali 
if the user sends `{ seen: false }` in this method, the message will be marked as unseen?
or a new ` unseen : boolean ` option will be added in future?

Reply:
Yes, the API support { seen: false }.
IMO markInAppMessageAs name could confuse the usee because it can be interpreted as it could mark as true/false.

Reply:
rename this method to `markMessageAs`